### DISCUSSION TITLE
Plugin developement

### DISCUSSION THREAD

**Discussion by hra7** [dragonflyoss/Dragonfly2#1062] [General]
hey I have developed the basic  plugin for dragonfly following your documentation . In documentation you have not specified abt the folder structure where it should be placed before building it into .so file ,can u pls specify it.

**Comment by jim3ma** [comment]
In docs/en/developer-guide/plugin.md, we build plugin with go module, so the path is not important.
Another way to build plugin with docker: https://github.com/dragonflyoss/Dragonfly2/blob/main/docs/en/developer-guide/plugin-builder.md
